doc/api/fs.md

```diff
@@ -2211,20 +2211,17 @@ changes:
 -->
 
 * `path` {string|Buffer|URL}
-* `atime` {integer}
-* `mtime` {integer}
+* `atime` {number|string|Date}
+* `mtime` {number|string|Date}
 * `callback` {Function}
 
-Change file timestamps of the file referenced by the supplied path.
-
-*Note*: The arguments `atime` and `mtime` of the following related functions
-follow these rules:
+Change the file system timestamps of the object referenced by `path`.
 
-- The value should be a Unix timestamp in seconds. For example, `Date.now()`
-  returns milliseconds, so it should be divided by 1000 before passing it in.
-- If the value is a numeric string like `'123456789'`, the value will get
-  converted to the corresponding number.
-- If the value is `NaN`, `Infinity` or `-Infinity`, an Error will be thrown.
+The `atime` and `mtime` arguments follow these rules:
+- Values can be either numbers representing Unix epoch time, `Date`s, or a
+  numeric string like `'123456789.0'`.
+- If the value can not be converted to a number, or is `NaN`, `Infinity` or
+  `-Infinity`, a `Error` will be thrown.
```

Comment:
> @refack Sorry, I meant a nit, still "an", not "a")

Ack. But I'm not pushing a fix yet, so the `string` conversation will stay open.